### PR TITLE
{bio}[foss/2023a] AFNI v24.0.02

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

Requires:
- [x] PR #19655

### CODE REVIEW

**Comment by sassy-crick** [comment]
This is a follow-up of PR #17309. In both PRs the installation has been improved so no files should end up in directories like `.local` etc. where they clearly should not be. 
Also, from what I can tell, all scripts are in the bin directory now, or a subdirectory thereof. 
Finally `afni_system_check.py -check_all` gives only a few warning. One which I could not work out is this one:
```
    3dMVM                : FAILURE
        Error in if (!length(file) || file == "") stop("empty file/url name") : 
          missing value where TRUE/FALSE needed
        Calls: source
        Execution halted
```
My hunch is it might have something to do with the used `tcsh` as it works in PR #17309 . If somebody has an idea, please contribute. 
Both `afni` and `somo` seem to work.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/a/AFNI/AFNI-24.0.02-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    ('Python', '3.11.3'),
+    ('X11', '20230603'),
+    ('motif', '2.3.8'),
+    ('R', '4.3.2'),
+    ('PyQt5', '5.15.10'),
+    ('expat', '2.5.0'),
+    ('libpng', '1.6.39'),
+    ('libjpeg-turbo', '2.1.5.1'),
+    ('GSL', '2.7'),
+    ('GLib', '2.77.1'),  # must match version used in Qt5 (via PyQt5)
```
Comment (Micket) [inline easybuild/easyconfigs/a/AFNI/AFNI-24.0.02-foss-2023a.eb:35]:
```suggestion
    ('GLib', '2.77.1'),
```

**Review by Micket — APPROVED** [review]
lgtm

**Pull request merged**